components/I18nFormatters.js

```diff
@@ -8,6 +8,7 @@ export const getI18nLink = linkProps => chunks =>
   <StyledLink {...linkProps}>{linkProps?.children || chunks}</StyledLink>;
 export const I18nBold = chunks => <strong>{chunks}</strong>;
 export const I18nItalic = chunks => <i>{chunks}</i>;
+export const I18nUnderline = chunks => <u textDecoration="underline">{chunks}</u>;
```

Comment:
Great addition to the formatters, however it seems like the tag itself is enough? `textDecoration` doesn't seem like a standard HTML attribute

```suggestion
export const I18nUnderline = chunks => <u>{chunks}</u>;
```